client/signup/steps/site-information/index.jsx

```diff
@@ -206,10 +218,17 @@ export default connect(
 			! isBusiness && includes( ownProps.informationFields, 'title' )
 				? [ 'title' ]
 				: ownProps.informationFields;
+		const shouldFetchVerticalData =
+			ownProps.showSiteMockups &&
+			getSiteType( state ) === 'business' &&
```

Comment:
We probably don't need to check for the `siteType` here as we can rely on the `showSiteMockups` property, which is explicitly set for each step:

https://github.com/Automattic/wp-calypso/blob/33a03b0c475df7cf9b8172b1af27b2bda76ddd25/client/signup/config/steps-pure.js#L547

Reply:
Actually, I wrote that without noticing the `site_type=business` param in the URL. 

I'm still pretty confident we don't need the check, it's okay to leave in for now I guess.

Reply:
Thanks for pointing that out. I have a feeling that I <i>might</i> need it for the following landing pages issue, so I'll leave it this way for now. If it ends up not needed, I'll remove it in the upcoming PR.